Sources/Chat/NetworkingInteractor.swift

```diff
@@ -0,0 +1,112 @@
+
+import Foundation
+import Combine
+import WalletConnectRelay
+import WalletConnectUtils
+
+class NetworkingInteractor {
```

Comment:
Is it possible to reuse `NetworkInteractor` from sign?

Reply:
that is very good question.

I am afraid that for now it is more difficult to make it generic. And I already recognises some differences 

Sign also depends to Sign WCrequests now - and few more

That is definitely something to consider but I think it can be time consuming and not worth it now